shared/src/legalStructure.ts

```diff
@@ -132,6 +132,22 @@ export const LegalStructures: LegalStructure[] = [
       "dbaName",
     ]),
   },
+  {
+    id: "foreign-nonprofit",
```

Comment:
[pebble] Why is this a new entry here? We don't have a an entry for foreign-limited-liability-company or something. 

My understanding is that this list is used to supply the values of the legal structure radio buttons. If a user is onboarded as Poppy, we definitely don't want them to be able to select "Foreign Nonprofit" in their business structure. Why does this entry exist?

Reply:
We experienced buggy behavior with the `Suffix Dropdown` and found that this approach helped , but we had to filter out the foreign-nonprofit so that it doesn't appear. I'd love to find a different approach/hear suggestions.

Reply:
I think that since you defined `foreign-nonprofit` as a FormationLegalType, you will need to supply a value to the `BusinessSuffixMap` if the suffix tests aren't working?

Reply:
If you can't figure it out, we can chat. I don't think that adding it to this array of Legal Structures is a valid approach because it violates our domain boundaries.

For foreign legal structures, we don't think of them as a different LegalStructure type as their non-foreign counterparts. That's why there aren't any other `foreign-x` in this array. We don't want to break that pattern